docs/en/Web-Interface.md

```diff
@@ -62,7 +62,7 @@ The configuration page has two sections:
     made to a file not prefixed by a path in the white list. Only whitelisted files will be stored
     in Alluxio.
 
-## Browse File System Page
+## Browse
```

Comment:
I meant you should keep "Browse File System Page" here.
And update line 67:
When selecting the "Browse File System" tab -> When selecting the "Browse" tab